### PR TITLE
Link to blog, new countdown timer, layout changes, FAQ, responsiveness improvements

### PR DESCRIPTION
**Pull request by PradyumnaShome**

### CODE REVIEW

**Comment by PradyumnaShome** [comment]
- Makes the site **much** more responsive
- Adds the FAQ section
- Adds link to blog
- Added responsive countdown timer, with animations
- Better support for some aspect ratios including larger phones, iPads and iPad pros
- Pushed the image back down, as having the countdown timer on one row creates more space

File: source/index.jsx [hunk 1]
```diff
@@ -44,11 +45,12 @@ render(
     <Router>
         <div>
             <Route exact path="/" component={Home}/>
-            <Route path="/sponsor" component={SponsorLetter} />
```
Comment (modnikhil) [inline source/index.jsx]: Don't remove the sponsor route, that's where our sponsor tiers are located

File: source/index.jsx [hunk 2]
```diff
@@ -44,11 +45,12 @@ render(
...
-            <Route path="/startup" component={StartupLetter} />
```
Comment (modnikhil) [inline source/index.jsx]: See sponsor route comment

File: source/index.jsx (hunk 1 above)
Comment (PradyumnaShome) [reply source/index.jsx]: At the time, they were unused and empty, which was why I removed them.

File: source/components/home/navigation/navigation.js [hunk 3]
```diff
@@ -37,6 +37,8 @@ class Navigation extends Component {
                         <NavItem onClick={() => this.scrollTo('Speakers')} eventKey={2} >Past Speakers</NavItem>
                         <NavItem onClick={() => this.scrollTo('Sponsors')} eventKey={3} >Past Sponsors</NavItem>
                         <NavItem onClick={() => this.scrollTo('Footer')} eventKey={4} >Contact Us</NavItem>
+                        <NavItem onClick={() => location.href="https://blog.reflectionsprojections.org"} eventKey={5}>Blog</NavItem>
```
Comment (modnikhil) [inline source/components/home/navigation/navigation.js]: Can you swap the order a little and make Contact Us the last menu item? Blog -> FAQ -> Contact Us

File: source/components/home/countdown/countdown.js [hunk 4]
```diff
@@ -1,40 +1,70 @@
...
-    
+
     constructor() {
         super();
     }
 
-    componentWillMount(){
-      var countDownDate = new Date("Sep 17, 2018 00:00:00").getTime();
-        var x = setInterval(function() {
+    componentWillMount() {
```
Comment (modnikhil) [inline source/components/home/countdown/countdown.js]: Is there any way to make the circles animated? On first glance the way it's designed makes it look like the circle indicated how much time is left until another second/minute/hour/day elapses

**Comment by PradyumnaShome** [comment]
I think this is ready for merging, @modnikhil @ahmedammar841.

**Comment by PradyumnaShome** [comment]
Closing in favour of #51.

It appears that the Contact Us change and removed routes change do not show up in #51, but rest assured I will add once that is merged in.

@modnikhil

**Pull request closed without merging**